app/main.ipynb

```diff
@@ -185,24 +217,28 @@
+     },
+     "execution_count": 246,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
-    "# write your code for task 6 here"
+    "employees_info.groupby(\"age\")[\"education\"].describe()"
```

Comment:
It must be statistics of age for each type of education, swap `age` and `education`